projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionResultManager.java

```diff
@@ -109,7 +110,7 @@ private static ExecutionTerminationStatus getTerminationStatus(String podTermina
     * @param terminationStatus termination status based on the K8S Pod termination status
     * @param executionStatus execution status based on K8S Job status
     * @param terminationStatusString termination status returned from K8S Pod (e.g. "Success", "User error", etc.)
-    * @param terminationReason condition reason as reported by K8s Job (e.g. "DeadlineExceeded", "BackoffLimitExceeded", etc.)
+    * @param jobTerminationReason condition reason as reported by K8s Job (e.g. "DeadlineExceeded", "BackoffLimitExceeded", etc.)
     * @return if there is no termination message due to the missing K8S Pod
```

Comment:
You can add a comment for the new parameter.

Reply:
Comment added, thanks